scipy/stats/tests/test_distributions.py

```diff
@@ -371,6 +371,195 @@ def test_pdf_zero(self):
+            )
+
+        assert_allclose(gh.cdf(x), vals_R, atol=1e-9, rtol=1e-5)
+
+    def test_moments_r(self):
+        # test against R package GeneralizedHyperbolic
+        # sapply(1:4,
+        #    function(x) GeneralizedHyperbolic::ghypMom(
+        #        order = x, lambda = 2, alpha = 2,
+        #        beta = 1, delta = 1.5, mu = 0.5,
+        #        momType = 'raw')
+        # )
+
+        vals_R = np.array([
+            2.36848366948115, 8.4739346779246,
+            37.8870502710066, 205.76608511485
+            ])
+
+        lmbda = 2
+        alpha = 2
+        beta = 1
+        delta = 1.5
+        mu = 0.5
+
+        alpha_hat = alpha * delta
+        beta_hat = beta * delta
+
+        vals_us = [
+            stats.genhyperbolic(p=lmbda, a=alpha_hat, b=beta_hat,
+                                loc=mu, scale=delta).moment(i)
+            for i in range(1, 5)
+            ]
+
+        assert_allclose(vals_us, vals_R, atol=1e-9, rtol=1e-8)
+
+    def test_rvs(self):
+        # KS test
+        # p = 1, alpha = 1, beta = 0
+        gh = stats.genhyperbolic(p=1, a=1, b=0)
+        _, p = stats.kstest(gh.rvs(size=1500, random_state=1234), gh.cdf)
+
+        assert_equal(p > 0.05, True)
+
+    def test_pdf_t(self):
+        # Test Against T-Student with 1 - 30 df
+        df = np.linspace(1, 30, 20)
+        # in principle alpha should be zero in practice for big lmbdas
+        # alpha cannot be too small else pdf does not integrate
+        alpha = np.float_power(df, 2)*np.finfo(np.float32).eps
+        beta = 0
+        mu = 0
+        delta = np.sqrt(df)
+        alpha_hat = alpha * delta
+        beta_hat = beta * delta
+        gh = stats.genhyperbolic(
+            p=-df/2,
+            a=alpha_hat,
+            b=beta_hat,
+            loc=mu,
+            scale=delta
+            )
+        x = np.linspace(gh.ppf(0.01), gh.ppf(0.99), 100)[:, np.newaxis]
+        assert_allclose(
+            gh.pdf(x), stats.t.pdf(x, df),
+            atol=1e-8)
+
+    def test_pdf_laplace(self):
+        # Test Against Laplace with location param [-10, 10]
+        loc = np.linspace(-10, 10, 20)
+        # in principle delta should be zero in practice for big loc delta
+        # cannot be too small else pdf does not integrate
+
+        lmbda = 1
+        alpha = 1
+        beta = 0
+        delta = np.finfo(np.float32).eps
+
+        alpha_hat = alpha * delta
+        beta_hat = beta * delta
+
+        gh = stats.genhyperbolic(
+            p=lmbda,
+            a=alpha_hat,
+            b=beta_hat,
+            loc=loc,
+            scale=delta
+            )
+        # ppf does not integrate for scale < 5e-4
+        # therefore using simple linspace to define the support
+        x = np.linspace(-20, 20, 100)[:, np.newaxis]
+        assert_allclose(
+            gh.pdf(x), stats.laplace.pdf(x, loc=loc, scale=1),
+            atol=1e-9)
+
+    def test_pdf_norminversegauss(self):
+        # Test Against NIG
```

Comment:
comment not needed

Reply:
Extended the comment so it is more clear. Leavimng it there for coherence with above test_ functions